notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb

```diff
@@ -1,1330 +1,1359 @@
 {
```

Comment:
<p>Explain the importer step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>done</p>